pydicom/dataset.py

```diff
@@ -1365,3 +1397,60 @@ def __eq__(self, other):
             return self_elem == other_elem and self.__dict__ == other.__dict__
 
         return NotImplemented
+
+
+def check_and_fix_file_meta_info(file_meta, enforce_standard=True):
+    """Checks the File Meta Information elements in `file_meta` and
```

Comment:
`validate_file_meta`?

Reply:
Ok, `validate_file_meta` is probably better, given that correction is optional. Done.